src/models/Option.js

```diff
@@ -28,17 +29,24 @@ export default class Option {
     if (event.status !== EVENT_STATUS.ARBITRATION) {
       const totalBalance = _.sum(event.roundBets);
       this.percent = totalBalance === 0 ? totalBalance : _.round((this.amount / totalBalance) * 100);
+      this.userPercent = this.amount === 0 ? this.amount : _.round((event.userRoundBets[i] / this.amount) * this.percent);
     } else {
       this.isPrevResult = event.currentResultIndex === i;
+      this.value = this.isPrevResult ? event.previousRoundBets[i] : this.amount;
       this.maxAmount = event.status === EVENT_STATUS.ARBITRATION
         ? event.consensusThreshold - decimalToSatoshi(this.amount) : undefined;
 
-      const threshold = this.isPrevResult ? 0 : event.consensusThreshold;
-      this.percent = threshold === 0 ? threshold : _.round((this.amount / threshold) * 100);
+      const threshold = this.isPrevResult ? event.previousConsensusThreshold : event.consensusThreshold;
+      this.percent = this.isPrevResult ? 100 : _.round((this.amount / threshold) * 100);
+      if (this.isPrevResult) {
```

Comment:
nit: inconsistent here with the way you set the values. `threshold` and `percent` using ternaries but checking the same `isPrevResult` while this one is an if statement. suggest to stick to one or the other

Reply:
use if statement here since linter requires not to nested ternaries

Reply:
check the new one, it becomes more complex to handle the result set. And do i need to make all of the ternary to if statement?

Reply:
I was just pointing out the code looks inconsistent by doing checks two diff ways:
```
myVar = isPrevResult ? a : b
myVar2 = isPrevResult ? a : b
if (isPrevResult) {
  a
} else {
  b
}
```

suggesting to do something like:
```
myVar
myVar2
if (isPrevResult) {
  myVar = a
  myVar2 = a
} else { 
  myVar = b
  myVar2 = b
}
```

not a huge deal though which is why i put nit. if you feel its ok, then just leave it.